R/qsys_sge.r

```diff
@@ -44,6 +54,10 @@ PBS = R6::R6Class("PBS",
         initialize = function(..., template=getOption("clustermq.template", "PBS")) {
             super$initialize(..., template=template)
         }
+    ),
+    
+    private = list(
+      set_job_id = function() {private$job_id = private$qsub_stdout[1]}
```

Comment:
According to the documentation of Torque, this should also go into that class

Reply:
Torque now inherits PBS